snowflake/udfs/udfs/traffic_drop/create_function.sql

```diff
@@ -0,0 +1,22 @@
+create or replace function traffic_drop_%VERSION%(
+    jobType STRING(20),
+    detectionID STRING(40),
+    endpoint STRING,
+    direction STRING,
+    date DATE,
+    dropNumber NUMBER(20, 0)
+)
+returns table ( jobType STRING(20),
+                detectionID STRING(40),
+                timeCreated TIMESTAMP_NTZ,
+                endpoint STRING,
+	            direction STRING,
+	            avgDrop NUMBER(25, 5),
+	            stdevDrop NUMBER(25, 5),
+                anomalyDropDate DATE,
+	            anomalyDropNumber NUMBER(20, 0))
```

Comment:
maybe you could fix the indentation here

Reply:
Thanks, fixed.